packages/sdk-react/src/components/payables/PayableDetails/PayableDetails.tsx

```diff
@@ -196,12 +198,14 @@ const PayableDetailsBase = ({
                   optionalFields={optionalFields}
                   lineItems={lineItems}
                   payableDetailsFormId={payableDetailsFormId}
+                  ocrRequiredFields={ocrRequiredFields}
                 />
               ) : (
                 payable && (
                   <PayableDetailsInfo
                     payable={payable}
                     optionalFields={optionalFields}
+                    ocrRequiredFields={ocrRequiredFields}
```

Comment:
@saladNights, do you think it might make more sense to use `Theme` and customize `PayableDetailsInfo` here?
I wouldn't really want to drag this customization into the public `PayableDetails` interface, especially since it would be impossible to customize this for Drop-In or Iframe.

Reply:
that's a good idea. It may not be perfect since it's a mix of concepts, but we already have mechanism in place.

Reply:
I vote for customization via theme.

Moreover, I would move the `optionalFields` in the future major version well.